### PR TITLE
Added ability to choose square or non-square icons.

### PR DESCRIPTION
**Pull request by ghost**
Addresses https://github.com/backdrop-contrib/on_the_web/issues/21.

### CODE REVIEW

File: on_the_web.module [hunk 1]
```diff
@@ -184,11 +184,18 @@ function on_the_web_display_block($delta, $settings = array(), $contexts = array
       $size = $config->get('size');
       $classes = array('on-the-web', 'otw-' . $size, 'otw-' . $service);
 
+      $icon = $info['fa-icon'];
+      if ($config->get('type') == 'font' && $config->get('square') == TRUE) {
+        if (isset($info['fa-icon-square'])) {
+          $icon = $info['fa-icon-square'];
```
Comment (jenlampton) [inline on_the_web.module:190]: Nice, I like the graceful fall back to non-square here :)

File: on_the_web.module [hunk 2]
```diff
@@ -255,39 +262,44 @@ function on_the_web_get_services() {
...
+      'fa-icon' => 'fa-youtube',
     ),
     'flickr' => array(
       'name' => 'Flickr',
       'fa-icon' => 'fa-flickr',
     ),
     'linkedin' => array(
       'name' => 'LinkedIn',
-      'fa-icon' => 'fa-linkedin-square',
+      'fa-icon' => 'fa-linkedin',
```
Comment (jenlampton) [inline on_the_web.module:302]: I think we might me missing a `fa-linkedin-square` here, http://fontawesome.io/icon/linkedin-square.

**Comment by jenlampton** [comment]
This PR looks great @BWPanda! The only thing I think we need to add is the square version of the linked in icon: http://fontawesome.io/icon/linkedin-square

**Comment by jenlampton** [comment]
I've merged this and added a follow up commit to add two missing icons: https://github.com/jenlampton/on_the_web/commit/739887ceb4d3d8affc0cc2c91d04c7e7dabcd0bd

**Comment by ghost** [comment]
Nice, thanks! Re. LinkedIn, I think I was looking at the new v5 icons: https://fontawesome.com/icons?d=gallery&q=linkedin&m=free

**Pull request merged**